docs/src/main/asciidoc/security-jwt.adoc

```diff
@@ -807,6 +807,19 @@ Finally both the confidentiality and integrity of the claims can be further enfo
 
 SmallRye JWT provides an API for securing the JWT claims using all of these options.
 
+== Maven dependency
+
+[source,xml]
+----
+<dependency>
+  <groupId>io.smallrye</groupId>
+  <artifactId>smallrye-jwt-build</artifactId>
+  <version>${smallrye.jwt.version}</version>
```

Comment:
Is the version still necessary here?

Reply:
@gastaldi Sorry, missed this question - I don't know - it is not in Quarkus `bom/application/pom.xml` dependencies yet as at the moment `quarkus-smallrye-jwt`/`quarkus-oidc` depend on it indirectly.
I can update this section later, as part of the fix for #13499, to include `io.quarkus/quarkus-smallrye-jwt-build` (no version) ?

Reply:
I think adding it to the BOM should be done in the same commit.

Reply:
So you can remove the version now from here now as it will automatically come with the BOM.

Reply:
@gsmet, yeah, sorry, I realized it once I switched my laptop off and went for a walk :-), sorry :-)